app/views/assessments/index.html.erb

```diff
@@ -6,46 +6,55 @@
 <% end %>
 
 <% if @is_instructor %>
-<h2>Instructor Actions</h2>
+<div class="section">
+  <h3 class="section-title"><span class="white">Instructor Actions</span></h4>
   <%= link_to 'Install Assessment',
                     {:action=>"installAssessment"},
-                    {:title=>"Install Assessment", :class => "btn" } %>
-  <% end %>
-
-<h2>Assignments</h2>
-<div class="rolodex">
-<% @course.assessment_categories.each do |cat| %>
-  <% asmts = @course.assessments_with_category(cat, @cud.student?) %>
-  <% if asmts.any? %>
-  <div class="card">
-    <h1><a href=""><%= cat %></a></h1>
-    <div class="content">
-      <ul>
-      <% asmts.each do |asmt| %>
-        <li>
-          <%= link_to asmt.display_name, course_assessment_url(@course, asmt) %><%= "*" unless asmt.released? %>
-          <span class="pull-right shortcuts">
-            <a class="goto-icon-small" href="<%= course_assessment_path(@course, asmt) %>"><%= image_tag "go-new.png", :height => 22, :alt=>"Go to assessment" %></a>
-          </span>
-        </li>
+                    {:title=>"Install Assessment", :class => "waves-effect waves-light btn btn-large red darken-4" } %>
+<% end %>
+
+<div class="section">
+  <h3 class="section-title"><span class="white">Assignments</span></h3>
+
+  <div class="row">
+    <% @course.assessment_categories.each_with_index do |cat, idx| %>
+      <% if idx % 3 == 0 %>
+        <div class="clearfix hide-on-small-only"></div>
       <% end %>
-      <% if asmts.length == 0%>
-       <div><i>No assignments in this category.</i></div>
+      <% asmts = @course.assessments_with_category(cat, @cud.student?) %>
+      <% if asmts.any? %>
+        <div class="col s12 m4">
+          <div class="card hoverable">
+            <div class="card-content red darken-4">
+              <span class="card-title white-text"><%= cat %></span>
+            </div>
+
+            <div class="collection red darken-4" style="border-bottom: 0">
+              <% asmts.each do |asmt| %>
+                <%= link_to(course_assessment_url(@course, asmt),
+                            {:class => "collection-item grey-text text-darken-4"}) do %>
+                    <%= asmt.display_name %>
+                    
+                    <% if !asmt.released? %>
+                      <span class="new badge blue darken-4" data-badge-caption="unreleased"></span>
```

Comment:
Can this badge link to the page where you can release the assessment? So the person won't have to go the normal way, but go straight there. I think it's the edit handin page `/courses/AutoPopulated/assessments/homework0/edit/handin`